go/example/example.go

```diff
@@ -1,12 +1,21 @@
 package main
 
 import (
-	"../uaparser" // You could change this to a github repo as well
 	"fmt"
+	"io/ioutil"
+	"os" // You could change this to a github repo as well
+
+	"../uaparser"
 )
 
 func main() {
-	testStr := "Mozilla/5.0 (Macintosh; U; Intel Mac OS X 10_6_3; en-us; Silk/1.1.0-80) AppleWebKit/533.16 (KHTML, like Gecko) Version/5.0 Safari/533.16 Silk-Accelerated=true"
+	testBytes, err := ioutil.ReadAll(os.Stdin)
```

Comment:
Honestly I don't remember...probably I've changed that for testing purpose (piping user agents from a text file) or api response.